### PR TITLE
Changing values

### PR DESCRIPTION
**Pull request by nikhil364**

### CODE REVIEW

File: charts/avatar-creation/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.3 #0.1.2
+version: 0.1.4 #0.1.3 #0.1.2
```
Comment (edalford11) [inline charts/avatar-creation/Chart.yaml:18]: Can you remove the commented out versions?

File: charts/avatar-creation/values.yaml [hunk 2]
```diff
@@ -57,6 +57,7 @@ env:
   MEASURE_HOST: measure-engine
   BUCKET: avatar-creation-dev
   REGION: us-west-2
+  AVATAR_CONTROLLER_HOST: avatar-api
```
Comment (edalford11) [inline charts/avatar-creation/values.yaml:60]: Shouldn't the env var name be `AVATAR_API_HOST` ?

**Review by edalford11 — CHANGES_REQUESTED** [review]

File: charts/avatar-creation/values.yaml (hunk 2 above)
Comment (nikhil364) [reply charts/avatar-creation/values.yaml:60]: i would have to make that change in code, its fine the changed value should work

Comment (nikhil364) [reply charts/avatar-creation/values.yaml:60]: I can add this change later, the repo name is also avatar-controller too, I would have to change that too

File: charts/avatar-creation/Chart.yaml (hunk 1 above)
Comment (nikhil364) [reply charts/avatar-creation/Chart.yaml]: removed

File: charts/avatar-creation/values.yaml (hunk 2 above)
Comment (edalford11) [reply charts/avatar-creation/values.yaml:60]: You should change it everywhere. We can't allow these kinds of these to get in. This is technical debt. Naming is very important as to not confuse people.

**Pull request closed without merging**